site2/docs/deploy-docker.md

```diff
@@ -2,51 +2,163 @@
 id: deploy-docker
 title: Deploy a cluster on Docker
 sidebar_label: "Docker"
----
-
-To deploy a Pulsar cluster on Docker, complete the following steps:
-1. Deploy a ZooKeeper cluster (optional)
-2. Initialize cluster metadata
-3. Deploy a BookKeeper cluster
-4. Deploy one or more Pulsar brokers
-
-## Prepare
-
-To run Pulsar on Docker, you need to create a container for each Pulsar component: ZooKeeper, BookKeeper and broker. You can pull the images of ZooKeeper and BookKeeper separately on [Docker Hub](https://hub.docker.com/), and pull a [Pulsar image](https://hub.docker.com/r/apachepulsar/pulsar-all/tags) for the broker. You can also pull only one [Pulsar image](https://hub.docker.com/r/apachepulsar/pulsar-all/tags) and create three containers with this image. This tutorial takes the second option as an example.
+—
```

Comment:
This is an obvious syntax error that fails the site build.

Please be careful when reviewing...Always previewing your changes helps.

Reply:
I made a suggestion to modify the description template. I think we should process this to ensure that rather than hope that reviewers and authors are always right.

Reply:
No. The community runs by meritocracy so reviewers should be careful. It's not irreversible to make mistake like this and I'm not blaming you here. But send a notice that you should follow the process. Adding more description hints you to follow the process also but I believe few people really read the whole description.

Reply:
Shouldn't there be a sanity check CI job for any documentation change in the repo if full pipeline is too expensive?